htdocs/core/class/fiscalyear.class.php

```diff
@@ -195,7 +195,6 @@ public function update($user)
 		$sql .= ", date_start = '".$this->db->idate($this->date_start)."'";
 		$sql .= ", date_end = ".($this->date_end ? "'".$this->db->idate($this->date_end)."'" : "null");
 		$sql .= ", statut = '".$this->db->escape($this->statut?$this->statut:0)."'";
-		$sql .= ", datec = " . ($this->datec != '' ? "'".$this->db->idate($this->datec)."'" : 'null');
```

Comment:
The field datec is a mandatory field into table. Removing it will create an error.
May be your database is not up to date ?
We can add a migrate instruction to match the definition of table for such case.

Reply:
@eldy This field coming from v3.8... My database is up to date.
We are in update function of class fiscal year, not create function.
datec is defined by $now (And it's work) when fiscal year is created. After it's a modification, the field tms is updated.

Try to update a fiscal year with all version before 3.8 and you will have this error.